src/backend/joanie/settings.py

```diff
@@ -275,6 +275,20 @@ class Base(Configuration):
         environ_prefix=None,
     )
 
+    JOANIE_LCMS_BASE_URL = values.Value(
```

Comment:
This is netpicking I guess, but Richie is not a LCMS. We should probably use catalog ?